WordPressKit.podspec

```diff
@@ -2,7 +2,7 @@
 
 Pod::Spec.new do |s|
   s.name          = 'WordPressKit'
-  s.version       = '4.46.0'
+  s.version       = '4.46.1'
```

Comment:
@leandroalonso Is this the correct minor version you wanted me to bump it to? Or did you want me to make it something like `4.47.0-beta.1`?

Reply:
Let's take a look at the list of latest releases: https://github.com/wordpress-mobile/WordPressKit-iOS/releases

It should be `4.46.0-beta.3`

Reply:
@wargcm @leandroalonso FYI: I forgot to publish the [`4.46.0` stable release](https://github.com/wordpress-mobile/WordPressKit-iOS/releases/tag/4.46.0) for the `4.46.0` tag on Monday, so `4.46.0-beta.3`, while the correct guess based on what the Releases page of this repo showed at the time Leandro left the previous comment, is not the right value.

In the meantime, @ScoutHarris kindly shipped [`4.47.0-beta.1`](https://github.com/wordpress-mobile/WordPressKit-iOS/releases/tag/4.47.0-beta.1). If you need to update your `Podfile` to use the changes from this PR, please use `4.47.0-beta.1`.

Apologies for the confusion 😞

Reply:
No worries, thanks for letting me know!